system/doc/design_principles/distributed_applications.md

```diff
@@ -131,7 +131,19 @@ calling `application:start(myapp)` at all three nodes. It is then started at
 
 [](){: #dist1 }
 
-![Application myapp - Situation 1](assets/dist1.gif "Application myapp - Situation 1")
+```mermaid
+---
+title: Application myapp - Situation 1
+---
+
+stateDiagram
+    state myapp {
```

Comment:
I think it would make more sense if `myapp` is inside `cp1` instead of  the other way around.